.happy/terraform/modules/sfn/main.tf

```diff
@@ -211,21 +211,21 @@ resource "aws_sfn_state_machine" "state_machine_seurat" {
 EOF
 }
 
-resource "aws_sfn_state_machine" "state_machine_cxg_remaster" {
-  name     = "dp-${var.deployment_stage}-${var.custom_stack_name}-cxg-remaster-sfn"
+resource "aws_sfn_state_machine" "state_machine_cxg_remaster_v2" {
+  name     = "dp-${var.deployment_stage}-${var.custom_stack_name}-cxg-remaster-v2-sfn"
   role_arn = var.role_arn
 
   definition = <<EOF
 {
-  "StartAt": "CxgRemaster",
+  "StartAt": "CxgRemasterV2",
```

Comment:
code clarity suggestion: if we're not keeping the previous CxgRemaster step, is there any reason to name this CxgRemasterV2? What do you think of keeping the CxgRemaster naming convention and just changing the implementation? It adds an extra layer of maintenance to version it, and some confusion since we're not really versioning it such that we could still easily use or route to 'v1'

Reply:
still need to revert this back to cxg_remaster